staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/downloader.go

```diff
@@ -92,7 +53,7 @@ func etagFor(data []byte) string {
 // Download downloads openAPI spec from /swagger.json endpoint of the given handler.
 // httpStatus is only valid if err == nil
 func (s *Downloader) Download(handler http.Handler, etag string) (returnSpec *spec.Swagger, newEtag string, httpStatus int, err error) {
```

Comment:
I think in the future we could change this to also use the same downloader type we do but with another handler wrapper which does the retry at the old endpoint but it seemed out of scope for this PR

Reply:
Just added the comment above before reading this. Another PR looks good to me.